src/ocm/components/clusterDetail/utils.tsx

```diff
@@ -72,6 +72,7 @@ const getClusterResources = (cluster: Cluster, resourcePath: string): number =>
     )
     .map((host) => stringToJSON<Inventory>(host.inventory) || {})
     .map((inventory) => get(inventory, resourcePath, 0))
```

Comment:
Following change fixes all linting errors on that function

```suggestion
    .map((inventory) => get(inventory, resourcePath, 0) as number)
```

Reply:
I was not sure if the data could contain non-numbers.  I'll apply your suggestion, thanks!.